### PR TITLE
refactor: remove assert from common code (src/comon/k8stest/util.go)

### PR DESCRIPTION
**Pull request by rohan2794**
JIRA: https://mayadata.atlassian.net/browse/MQ-2747

### CODE REVIEW

**Comment by rohan2794** [comment]
bors try

**Comment by rohan2794** [comment]
bors try

**Comment by rohan2794** [comment]
bors try

**Comment by rohan2794** [comment]
bors try

**Comment by rohan2794** [comment]
bors try

File: src/common/k8sinstall/util.go [hunk 1]
```diff
@@ -365,6 +402,9 @@ func deleteNamespace() error {
 // objects, so that we can verify the local deploy yaml files are correct.
 func TeardownMayastor() error {
 	var cleaned bool
+	errors := make(chan error, 1)
```
Comment (chriswldenyer) [inline src/common/k8sinstall/util.go:405]: I think you will need to change 1 to len(mayastorYamlFiles) here, the number sets the buffer size in the channel and if more than one thread writes to the channel, the second thread will block, if the length is one.
Also on line 231.

File: src/common/k8stest/util_e2e_agent.go [hunk 2]
```diff
@@ -38,12 +42,11 @@ func EnsureE2EAgent() bool {
...
 
-	KubeCtlApplyYaml("e2e-agent.yaml", locations.GetE2EAgentPath())
+	err = KubeCtlApplyYaml("e2e-agent.yaml", locations.GetE2EAgentPath())
 
-	const sleepTime = 5
-	const duration = 60
-	count := (duration + sleepTime - 1) / sleepTime
-	ready := false
+	if err != nil {
+		return ready
```
Comment (chriswldenyer) [inline src/common/k8stest/util_e2e_agent.go:48]: suggestion: maybe explicitly return false here?

**Review by chriswldenyer — CHANGES_REQUESTED** [review]
LGTM except I think the channel length should be > 1

File: src/common/k8stest/util_e2e_agent.go (hunk 2 above)
Comment (rohan2794) [reply src/common/k8stest/util_e2e_agent.go]: Done

File: src/common/k8sinstall/util.go (hunk 1 above)
Comment (rohan2794) [reply src/common/k8sinstall/util.go]: Done

**Review by chriswldenyer — APPROVED** [review]
LGTM

**Review by payes — APPROVED** [review]

**Comment by rohan2794** [comment]
bors merge

**Pull request merged**